src/diagonal.works/b6/api/collections.go

```diff
@@ -632,6 +632,11 @@ func (a *ArrayTagCollection) Next() (bool, error) {
 var _ Collection = &ArrayTagCollection{}
 var _ Countable = &ArrayTagCollection{}
 
+type CollectionData struct {
```

Comment:
first i was like i ll just [2][]interface{} but then i was like this is more responsible of me to do but now i dont understand y we need so many different collection types / not just use smth like this / and do type assertions for different stuff / where we need to differentiate